src/app/static/src/tests/integration/load.itest.ts

```diff
@@ -16,9 +16,8 @@ describe("load actions", () => {
         const file = fs.createReadStream("../testdata/malawi.geojson");
         const formData = new FormData();
         formData.append('file', file);
-
-        await baselineActions.uploadShape({commit} as any, formData);
-        shape = (commit.mock.calls[1][0]["payload"] as ShapeResponse);
+        await baselineActions.uploadShape({commit, dispatch: jest.fn()} as any, formData);
+        shape = (commit.mock.calls[2][0]["payload"] as ShapeResponse);
```

Comment:
There is now an extra commit happening in this method (to invalidate survey/program when shape is updated), the actual shape response is the 3rd commit.